ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!---
+Hi! This template is here to help us collect optimal issue reports. Feel free to adapt it to your situation.
+First, check whether the issue has already been reported. If so, please add this report to that issue thread. If not, proceed to file this new issue.
+Next, please provide a one to four sentence summary of the issue. If you do your very best to be clear, concise and accurate here, other Rustaceans will be eager to assist. Also try to mention which GitHub labels this issue likely needs and/or what parts of Rust it relates to.
+--->
+
+## Code sample
+<!---
+Please craft a minimal/simple and self-contained code sample that (re)produces the issue. If you create a formal test case straight away, that'll be even more helpful.
```

Comment:
I do not disagree. It is pain to minimize other people’s crates to minimal test cases too. That being said see [this](https://github.com/rust-lang/rust/pull/31732#issuecomment-185449816).